infrastructure/main.tf

```diff
@@ -0,0 +1,101 @@
+provider "vault" {
+  //  # It is strongly recommended to configure this provider through the
+  //  # environment variables described above, so that each user can have
+  //  # separate credentials set in the environment.
+  //  #
+  //  # This will default to using $VAULT_ADDR
+  //  # But can be set explicitly
+  address = "https://vault.reform.hmcts.net:6200"
+}
+
+data "vault_generic_secret" "s2s_secret" {
+  path = "secret/${var.vault_section}/ccidam/service-auth-provider/api/microservice-keys/cmcLegalFrontend"
+}
+
+data "vault_generic_secret" "draft_store_secret" {
+  path = "secret/${var.vault_section}/cmc/draft-store/encryption-secrets/legal-frontend"
+}
+
+data "vault_generic_secret" "postcode-lookup-api-key" {
+  path = "secret/${var.vault_section}/cmc/postcode-lookup/api-key"
+}
+
+data "vault_generic_secret" "oauth-client-secret" {
+  path = "secret/${var.vault_section}/ccidam/idam-api/oauth2/client-secrets/cmc-legal"
+}
+
+locals {
+  aseName = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+}
+
+module "legal-frontend" {
+  source = "git@github.com:contino/moj-module-webapp.git"
+  product = "${var.product}-${var.microservice}"
+  location = "${var.location}"
+  env = "${var.env}"
+  ilbIp = "${var.ilbIp}"
+  is_frontend  = true
+  subscription = "${var.subscription}"
+
+  app_settings = {
+    // Node specific vars
+    NODE_ENV = "${var.env == "prod" ? "prod" : "dev"}"
```

Comment:
can you set the NODE_ENV to be "production" please:
https://dzone.com/articles/what-you-should-know-about-node-env